dev_guide/deployments.adoc

```diff
@@ -653,6 +653,32 @@ In the following example, each of `resources`, `cpu`, and `memory` is optional.
 <2> `*memory*` is in bytes; `256Mi` represents 268435456 bytes (`256 * 2 ^ 20`)
 ====
 
+However, if a quota has been defined for your project, one of the following two items is required:
+
+* a `resources` element must be set with an explicit `requests` of the form
+
+====
+
+[source,json]
+----
+{
+  "type": "Recreate",
+  "resources": {
+    "requests": { <1>
+      "cpu": "100m", 
+      "memory": "256Mi"
+    }
+  },
+}
+----
+<1> The request object contains the list of resources that correspond to the list of resources in the quota.
+====
+
+* your project also has a link:./limits.html[LimitRange] defined, where the defaults from the `LimitRange` will apply to pods
+created during the deployment process
+
+Otherwise, deploy pod creation will fail, citing the need to `make a non-zero request for memory since it is tracked by quota`.
```

Comment:
The error string is going to change shortly upstream, I would prefer we just say that the pod creation will fail citing a failure to satisfy quota.